### PR TITLE
Add custom module

### PR DESCRIPTION
**Pull request by reinaldoBloopark**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (agustinbloopark) [inline .idea/.gitignore:3]: This shouldn't be here

File: .idea/bloopark-workshop.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (agustinbloopark) [inline .idea/bloopark-workshop.iml:8]: very bad

File: addons/futsal/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+    'name': "Futsal",
+
+    'summary': """
+        Our futsal module""",
+
+    'description': """
+    """,
+
```
Comment (agustinbloopark) [inline addons/futsal/__manifest__.py:9]: Empty line? Why description is empty?

File: addons/futsal/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    'author': "bloopark systems GmbH & Co. KG",
+    'website': "http://www.bloopark.de",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/data/
+    # ir_module_category_data.xml
+    # for the full list
```
Comment (agustinbloopark) [inline addons/futsal/__manifest__.py:16]: what is this?

File: .idea/bloopark-workshop.iml (hunk 2 above)
Comment (omarbloopark) [inline .idea/bloopark-workshop.iml:8]: new line at the end! what is this file mean here ? bloopark-workshop.iml?

File: addons/futsal/models/res_partner.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+    _description = "Ground Owner"
+
+    has_ground = fields.Boolean('The owner has a ground')
+    ground_id = fields.Many2one(
+        'res.ground',
+        string='Ground'
```
Comment (agustinbloopark) [inline addons/futsal/models/res_partner.py:11]: Unnecessary string parameter

File: addons/futsal/security/groups.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<odoo>
+        <record id="group_owner" model="res.groups">
+            <field name="name">Owner</field>
+        </record>
+</odoo>
```
Comment (agustinbloopark) [inline addons/futsal/security/groups.xml:6]: new lion at the end

File: addons/futsal/security/ir.model.access.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+"access_manager","manager","model_res_ground","futsal.group_owner",1,1,1,1
```
Comment (agustinbloopark) [inline addons/futsal/security/ir.model.access.csv:2]: new lion at the end

File: addons/futsal/static/src/css/dealer-portal.css [hunk 8]
```diff
@@ -0,0 +1,10 @@
+.search-results a:hover {
+    text-decoration: none;
+}
+@media (min-width: 576px){
+.modal-dialog {
+    height: unset;
+    padding: 1.75rem 0;
+    margin-top: 5%;
+}
+}
```
Comment (agustinbloopark) [inline addons/futsal/static/src/css/dealer-portal.css:10]: why is this here?

File: .idea/inspectionProfiles/Project_Default.xml [hunk 9]
```diff
@@ -0,0 +1,56 @@
+<component name="InspectionProjectProfileManager">
```
Comment (omarbloopark) [inline .idea/inspectionProfiles/Project_Default.xml:1]: .idea folder shouldn't pushed here it's a local environment.

File: addons/futsal/static/src/css/main.css [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+}
+.search-box {
+    grid-column: 3;
+    grid-row: 1;
+    align-self: auto;
+
+}
+.employee_order_list {
+    padding-left: 10px;
+    padding-right: 10px;
```
Comment (agustinbloopark) [inline addons/futsal/static/src/css/main.css:36]: Beautiful css

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 11]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (omarbloopark) [inline .idea/inspectionProfiles/profiles_settings.xml:1]: same with .idea file

File: addons/futsal/static/src/css/order-form.css [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+td, th {
+  border: 1px solid #E9ECEF;
+  text-align: left;
+  padding: 8px;
```
Comment (agustinbloopark) [inline addons/futsal/static/src/css/order-form.css:24]: I love it

File: addons/futsal/static/src/js/dealer_order.js [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+            rpc.query({
+                route: '/dealer/search_customer_js',
+                params: {
+                    domain: domain,
+                    search: search,
+                }
+            }).then(function (data) {
+                // get values from controller to fill the list
+                if(data["id"]) {
+                    name = data["employee_name"]
```
Comment (agustinbloopark) [inline addons/futsal/static/src/js/dealer_order.js:39]: where is the copyright of this?

File: .idea/misc.xml [hunk 14]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (omarbloopark) [inline .idea/misc.xml:1]: we dont need this file !

File: .idea/modules.xml [hunk 15]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (omarbloopark) [inline .idea/modules.xml:1]: we dont need this file !

File: .idea/vcs.xml [hunk 16]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (omarbloopark) [inline .idea/vcs.xml:1]: we dont need this file !

File: addons/futsal/static/src/js/employee_order_list.js [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+                    id: value["id"], //order id will be assigned to html item
+                    url: value["url"], // to use in href
+                    edit_url: value["edit_url"],
+                    action_edit: value["action_edit"]
+                  })
+                }
+                var options = {
+                  valueNames: [ 'name', 'state', 'date_order' ],
+                  // Since there are no elements in the list,
+                  // this will be used as template.
```
Comment (agustinbloopark) [inline addons/futsal/static/src/js/employee_order_list.js:41]: cool js

File: addons/futsal/views/res_ground.xml [hunk 18]
```diff
@@ -0,0 +1,18 @@
...
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+</odoo>
```
Comment (agustinbloopark) [inline addons/futsal/views/res_ground.xml:18]: lion and zebras at the end

File: addons/futsal/__manifest__.py [hunk 19]
```diff
@@ -0,0 +1,31 @@
+{
+    'name': "Futsal",
+
```
Comment (omarbloopark) [inline addons/futsal/__manifest__.py:3]: remove empty lines here,

File: addons/futsal/__manifest__.py (hunk 4 above)
Comment (omarbloopark) [inline addons/futsal/__manifest__.py:16]: not needed!

File: addons/futsal/views/res_ground.xml (hunk 18 above)
Comment (MoustafaKh) [inline addons/futsal/views/res_ground.xml:18]: Please don't add the line at the end, we need to win +1

File: addons/futsal/models/res_ground.py [hunk 20]
```diff
@@ -0,0 +1,9 @@
+from odoo import _, api, fields, models
+
+
+class Ground(models.Model):
+    _name = 'res.ground'
```
Comment (omarbloopark) [inline addons/futsal/models/res_ground.py:5]: doc string missing

**Review by agustinbloopark — COMMENTED** [review]
zoo code

File: addons/futsal/models/res_partner.py [hunk 21]
```diff
@@ -0,0 +1,12 @@
+from odoo import _, api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
```
Comment (omarbloopark) [inline addons/futsal/models/res_partner.py:5]: doc string missing
